app/templates/main/500.html

```diff
@@ -9,6 +9,7 @@
     <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-l">Sorry, there is a problem with the service</h1>
         <p class="govuk-body">Try again later.</p>
+        <p class="govuk-body">Contact us at <a href="mailto:{{ config['CONTACT_EMAIL'] }}">{{ config['CONTACT_EMAIL'] }}</a>. Do not send your data return or any attachments using the help mail.</p>
```

Comment:
should we say '_if the problem persists_' contact us? Else it's a bit contradictory with the Try again later?

Reply:
Yes, I see your point. Do we need design sign-off for this?